### PR TITLE
Support for computed fields on abstract models

### PR DESCRIPTION
**Pull request by mobiware**
Fixes issue #12

### CODE REVIEW

**Comment by mobiware** [comment]
@jerch ok I've added more test cases to test dependencies with abstract models. Hopefully that's what you had in mind

File: computedfields/models.py [hunk 1]
```diff
@@ -55,8 +55,12 @@ def __new__(mcs, name, bases, attrs):
...
         cls = super(ComputedFieldsModelType, mcs).__new__(mcs, name, bases, attrs)
         if name != 'ComputedFieldsModel':
-            cls._computed_fields = computed_fields
-            mcs._computed_models[cls] = dependent_fields or {}
+            if hasattr(cls, '_computed_fields'):
+                cls._computed_fields.update(computed_fields)
+            else:
+                cls._computed_fields = computed_fields
+            if not cls._meta.abstract:
+                mcs._computed_models[cls] = dependent_fields or {}
```
Comment (jerch) [inline computedfields/models.py:63]: To make sense of these lines for me - this basically adds computed field definitions to `cls._computed_fields` over several `cls` constructions, and announcing the final definitions once it hits a non-abstract class, right?

I have not looked up django's dealing of abstract models in their metaclass, do you think there are hidden edge cases, where our metclass handling might break?

Comment (mobiware) [reply computedfields/models.py:63]: Yes that's what it does.

I don't know Django internals enough to tell whether there might be hidden edge cases

Comment (jerch) [reply computedfields/models.py:63]: Well the test covers most aspects regarding computed fields, so lets merge it. :smile_cat:
